### DISCUSSION TITLE
Absence of READ ONLY role in the LibreClinica

### DISCUSSION THREAD

**Discussion by SPAWARBCCC** [reliatec-gmbh/LibreClinica#416] [Ideas]
Currently we do not have READ ONLY role available in the LibreClinica EDC. Having of this role in EDC is important for Audit and Inspection purpose. Can @chae-reliatec @GerbenRienk @rhe-reliatec @oba-reliatec will take this request as product enhancement which will be added value to the LibreClinica EDC users. Thank you.

I would appreciate if other LibreClinica EDC users in this community provide their opinion on this topic. Thank you.

**Comment by oba-reliatec** [comment]
The monitor role cannot change or enter data, but it can file discrepancy notes, so it is almost, but not completely a READ ONLY role.

**Comment by toskrip** [comment]
I would support the idea of having simple reader role as a new feature request. Currently we are working on documentation how legacy roles are handled within codebase and once this is complete we should be able to estimate an afford needed to add additional role into the system. I often see a need for reader role while monitor is not really an alternative as typical reader should not be allowed to interfere with data monitoring workflow in prospective studies. Currently the only way of handling this is giving readers access to data exports out of EDC only (or when eCRF view necessary, study subject casebook print outs). It would be nice to have reader within the system itself.
